packages/components/board/src/board.css

```diff
@@ -1,13 +1,18 @@
 /* General........................................................ */
 
-.board-container {
+:host {
+	background-color: var(--ts-color-white);
 	border: var(--ts-border-width) var(--ts-border-type) var(--ts-border-color-default);
-	box-shadow: 1px 2px var(--ts-shadow-color);
 	border-radius: var(--ts-radius);
+	box-shadow: 0 2px 2px rgba(0, 0, 0, 0.1), 0 2px 8px rgba(0, 0, 0, 0.05);
```

Comment:
I don't think the border, border-radius and box-shadow should be override-able. Should we bring back the board-container class name and have them there?

Reply:
And maybe you can reuse the style in vars.css or if it's not available create a new one.

Reply:
if we put it inside the host, it will be clipped by "overflow:hidden" css rule. Without the "overflow:hidden" the inside content will not have rounded borders.
<img width="755" alt="Board without overflow hidden" src="https://user-images.githubusercontent.com/55530374/80959236-ace72980-8e06-11ea-8d86-d11987429939.png">